src/modules/AggregateRepository.js

```diff
@@ -158,11 +172,18 @@ export default class AggregateRepository {
     });
   }
 
-  find({ EntityClass, entityId, options }) {
+  find({ EntityClass, entityId, options = {} }) {
     const entity = this.createEntityInstance(EntityClass);
+    const { version } = options;
```

Comment:
`version` is addressed through a closure. And is not used on the level of its assignment. Perhaps, a better place for it would be inside the function that uses it individually.